src/main/java/com/github/xabgesagtx/bots/TelegramWebhookBotService.java

```diff
@@ -0,0 +1,21 @@
+package com.github.xabgesagtx.bots;
+
+import org.telegram.telegrambots.bots.TelegramWebhookBot;
+import org.telegram.telegrambots.meta.api.methods.updates.SetWebhook;
+
+/**
+ * This class is a blueprint to generate a Webhook Bot.
+ */
+public abstract class TelegramWebhookBotService extends TelegramWebhookBot {
```

Comment:
TelegramWebhookBot has two constructors.
1. no arguments
2. DefaultBotOptions

How about implementing both constructors, so people who wants to extend this class can still set the options?